evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -523,70 +523,6 @@ def test_graph_confusion_matrix_title_addition(X_y_binary):
     assert fig_dict['layout']['title']['text'] == 'Confusion matrix with added title text, normalized using method "true"'
 
 
-def test_get_permutation_importance_invalid_objective(X_y_regression, linear_regression_pipeline_class):
```

Comment:
There has been talk about splitting up `test_graphs.py`. I figured I would get a head start by creating `test_permutation_importance.py` and moving all of the permutation tests there.